pkg/products/marin3r/reconciler.go

```diff
@@ -198,6 +205,31 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 	return integreatlyv1alpha1.PhaseCompleted, nil
 }
 
+func (r *Reconciler) reconcileAlerts(ctx context.Context, client k8sclient.Client, installation *integreatlyv1alpha1.RHMI) (integreatlyv1alpha1.StatusPhase, error) {
+
+	cm := &corev1.ConfigMap{}
+	err := client.Get(context.TODO(), k8sclient.ObjectKey{Name: "ratelimit-config", Namespace: r.Config.GetNamespace()}, cm)
+	if err != nil {
+		logrus.Infof("didn't find ratelimit-config config map in %s", r.Config.GetNamespace())
+		return integreatlyv1alpha1.PhaseFailed, err
+	}
+
+	cmYamlData := yamlRoot{}
+	yaml.Unmarshal([]byte(cm.Data["kuard.yaml"]), &cmYamlData)
```

Comment:
```suggestion
	err = yaml.UnmarshalStrict([]byte(cm.Data["apicast-ratelimiting.yaml"]), &cmYamlData)
	if err != nil {
		return integreatlyv1alpha1.PhaseFailed, err
	}
```

I think this needs to be updated due to a changed in yaml name in :
* https://github.com/integr8ly/integreatly-operator/blob/master/pkg/products/marin3r/rateLimitService.go#L114

Also would be good to make this name a constant that is referenced in both these places and check for unmarshalling error :thinking: